src/Manager.php

```diff
@@ -94,7 +94,8 @@ protected function registerSmtp(): void
         $config = $this->getConfig();
 
         $this->mailer->isSMTP();
-
+        $this->mailer->SMTPAuth = true;
```

Comment:
This make sense to exist inside `if (isset($config['username']))` condition

Reply:
ok